chore: update project slogan (#4361)
GreptimeTeam/greptimedb: README.md

## Introduction

> **GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs**. You can gain real-time insights from Edge to Cloud at any scale.

Reviewer: Adds `trace in plan`?

Author: ```suggestion
**GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs** (also **trace** in plan). You can gain real-time insights from Edge to Cloud at any scale.
```

> **GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs** (also **trace** in plan). You can gain real-time insights from Edge to Cloud at any scale.

Author: ```suggestion
**GreptimeDB** is an open-source unified time-series database for **Metrics**, **Events**, and **Logs** (also **Traces** in plan). You can gain real-time insights from Edge to Cloud at any scale.
```

## Why GreptimeDB

Our core developers have been building time-series data platforms for years. Based on our best-practices, GreptimeDB is born to give you:

* **Unified all kinds of time series**

  GreptimeDB treats all time series as contextual events with timestamp, and thus unifies the processing of metrics and events. It supports analyzing metrics and events with SQL and PromQL, and doing streaming with continuous aggregation.

* **Cloud-Edge collaboration**

>   GreptimeDB can be deployed on ARM architecture-compatible Android/Linux systems as well as cloud environments from various vendors. Both sides run the same software, providing identical APIs and control plane, so your application can run at the edge or on the cloud without modification.

Reviewer: ```suggestion
  GreptimeDB can be deployed on ARM architecture-compatible Android/Linux systems as well as cloud environments from various vendors. Both sides run the same software, providing identical APIs and control planes, so your application can run at the edge or on the cloud without modification, and data synchronization also becomes extremely easy and efficient.
```

> * **Easy horizontal scaling**

Reviewer: Keep the title not changing. I think `cloud-native` is more proper here.

Author: ```suggestion
* **Cloud-native distributed database**
```
